content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -434,11 +434,14 @@ shared by the other messages of that session.
 
 In general, verification operates as follows:
 
-- Alice requests a key verification with Bob by sending an
-  `m.key.verification.request` event. This event indicates the verification
-  methods that Alice's client supports. (Note that "Alice" and "Bob" may in
-  fact be the same user, in the case where a user is verifying their own
-  devices.)
+- Alice requests a key verification with Bob by sending an key verification
+  request event.  If the verification is being requested in a room, this will
+  be an event with type `m.room.message` and `msgtype`
+  `m.key.verification.request`; if the verification is being requested using
+  to-device messaging, this will be an event with type
+  `m.key.verification.request`.  This event indicates the verification methods
+  that Alice's client supports. (Note that "Alice" and "Bob" may in fact be the
+  same user, in the case where a user is verifying their own devices.)
```

Comment:
It would be nice to link to the definitions of these things, but maybe if we were doing that, we'd want to do it for the whole section, and it becomes a major task.

Reply:
links added